frontend/libs/studio-components/src/components/StudioRecommendedNextAction/StudioRecommendedNextAction.tsx

```diff
@@ -0,0 +1,46 @@
+import { StudioButton } from '../StudioButton';
+import React from 'react';
+import classes from './StudioRecommendedNextAction.module.css';
+import { StudioCard } from '../StudioCard';
+
+export interface StudioRecommendedNextActionProps {
+  onSave: React.MouseEventHandler<HTMLButtonElement>;
+  saveButtonText: string;
+  onSkip: React.MouseEventHandler<HTMLButtonElement>;
+  skipButtonText: string;
+  title: string;
+  description: string;
+  hideSaveButton?: boolean;
+  children?: React.ReactNode;
```

Comment:
Should we make this prop mandatory instead? Is there any reason to use this component without children? 😊

Reply:
I see your good attation behind this! 👏  But I think that would be out-of-scope for this component. We do have a meeting next week about alerts and confirms, to figure out how to solve approve/decline challenge. 😊